LayoutTests/imported/w3c/web-platform-tests/navigation-api/focus-reset/autofocus.html

```diff
@@ -120,7 +120,6 @@
   assert_equals(document.activeElement, decoy, "Focus stays on the initially-focused button during the transition");
 
   decoy.disabled = true;
-  assert_equals(document.activeElement, document.body, "Disabling the initially-focused button temporarily resets focus to the body");
```

Comment:
I would prefer if you waited for the next rendering update for the focus fixup rule to happen here, rather than removing this. Though maybe this is not possible because it might race with `await finished;` below